src/theme/index.hbs

```diff
@@ -67,11 +67,11 @@
                 var theme = localStorage.getItem('mdbook-theme');
                 var sidebar = localStorage.getItem('mdbook-sidebar');
 
-                if (theme.startsWith('"') && theme.endsWith('"')) {
+                if (theme && theme.startsWith('"') && theme.endsWith('"')) {
                     localStorage.setItem('mdbook-theme', theme.slice(1, theme.length - 1));
                 }
 
-                if (sidebar.startsWith('"') && sidebar.endsWith('"')) {
+                if (sidebar && sidebar.startsWith('"') && sidebar.endsWith('"')) {
```

Comment:
These changes seem unrelated to the PR?

Reply:
they were null reference bugs I ran into while testing this PR.

Reply:
I would prefer unrelated changes to not be included.

Reply:
this is a bit different as I couldn't get the tests to pass without this.

Reply:
Which tests aren't passing? What kind of errors are you seeing? I don't believe there are any javascript tests, so it is not clear to me how this could have any impact on anything.

Reply:
Hmmm, trying to repro it I can't find it any more and I see there's a `try{ } catch (e) { }` around this whole thing, so it's possible I caught the exception in the debugger and mistook it for a bug when it was expecting the try/catch block to catch it and continue.  So I reverted these changes.